.agents/skills/dcc-nuxt-layers/references/nuxt-layer-feedback-control.md

```
@@ -0,0 +1,435 @@
+
+1. Go to **Settings** → **Developer settings** → **Personal access tokens** → **Fine-grained tokens**
+2. Click **Generate new token**
+3. Give it a descriptive name (e.g., "Feedback Control")
+4. Select only repositories and add you repository
+5. Select the following permissions:
+   - ✅ `contents` (read and write content)
+   - ✅ `issues` (read and write issues)
+5. Click **Generate token**
+6. Copy the token immediately (you won't see it again)
+
+** Production **
```

markdownlint-cli2 (0.23.1)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)